### DISCUSSION TITLE
Block Grid Layout Identification

### DISCUSSION THREAD

**Discussion by craigs100** [umbraco/Umbraco-CMS#13669] [Features and ideas]
Enjoying building out V11 sites with the Block Grid Editor. But as I dip in and out of V7 sites that are using the old Grid Editor I notice that it's far easier for an editor to understand which layout to apply because the icons to add a layout automatically display the columns, as shown below:
![image](https://user-images.githubusercontent.com/1143883/212081173-875d75d7-09a2-405d-9070-0cc048c6fcd3.png)

My question is, would it be possible to do have the Layout Blocks show their column definitions in the block's instead of, or in addition to, the icon?

![Screenshot from 2023-01-12 17-06-25-1](https://user-images.githubusercontent.com/1143883/212133586-caecd8b9-30a8-495b-bb2e-a51b3e62fdb1.png)

**Comment by skttl** [comment]
You can use the block thumbnail for this https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/block-grid-editor#catalogue-appearance

**Reply by craigs100** [reply]
Thanks for that. All now done :)

![Screenshot from 2023-01-13 14-26-41](https://user-images.githubusercontent.com/1143883/212343611-fc27a712-70fd-4986-9f1b-a80e99170c95.png)

The InkScape file is attached if anyone wants it. Just move the white bar around ;)

![block_layout_icons](https://user-images.githubusercontent.com/1143883/212336690-67839a5a-83d8-431a-a96e-9ebde0ed1a4a.svg)
